src/anonymizer/anonymizer.config.ts

```diff
@@ -9,6 +9,8 @@ const anonymizerConfig = {
   anonymizeInternalEmailUsername: toBoolean(true),
   enableInternalEmailPlusAddressing: toBoolean(true),
   enableExternalEmailPlusAddressing: toBoolean(false),
+  extractDomains: toBoolean(false),
+  extractDomainsWhitelist: toArray(),
```

Comment:
Whitelist should contain default values (`zoom.us,meet.google.com`) as stated in the docs.